packages/mobx-state-tree/src/utils.ts

```diff
@@ -407,7 +407,7 @@ export function isTypeCheckingEnabled() {
  * @hidden
  */
 export function devMode() {
-    return process.env.NODE_ENV !== "production"
+    return (typeof process === "undefined" || (process && process.env && process.env.NODE_ENV !== "production"));
```

Comment:
Note that this will bring MST into dev mode if process is _not_ defined. I'd expect 

`typeof process !== "undefined"  && process && process.env && process.env.NODE_ENV !== "production"`

Reply:
Hi Michel,

My idea was that the debug umd build _should_ be dev mode by default. By using your approach, it will result in production by default and the developer using the umd debug build should then (still) explicitly set process.env.NODE_ENV to enable dev mode. I think this would be confusing for developers (like me) that are not using node.js based tooling (and not using React).

However, I did a quick test and it appears that the minified umd version will _also_ be dev mode. That should not be the case. I think that is where your comment is coming from.

I wonder what your thoughts are on this. If you agree with me, perhaps you know a good method of implementing the above behavior. Thanks!